### PR TITLE
Updating Browser Upgrade messages

### PR DESCRIPTION
**Pull request by ezequielc**
@alex-deaconu Please review. Thanks!

### CODE REVIEW

File: src/locales_json/en/displays-app.json [hunk 1]
```diff
@@ -144,15 +144,14 @@
...
       "browser": {
-        "installed": "Installed Browser:",
+        "installed": "Browser Version:",
         "recommended": "Recommended Browser:",
         "upgrade": {
           "autoUpgrade": "Auto Upgrade",
           "current": "Current",
-          "name": "Player Upgrade:",
+          "name": "Browser Upgrade:",
           "previous": "Previous",
```
Comment (alex-deaconu) [inline src/locales_json/en/displays-app.json]: @ezequielc I think we should remove the `current` & `previous` options as they are no longer needed. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc looks good. 1 comment re: cleanup. Thanks.

File: src/locales_json/en/displays-app.json (hunk 1 above)
Comment (ezequielc) [inline src/locales_json/en/displays-app.json]: @alex-deaconu Yes, missed those. Thanks!

**Comment by alex-deaconu** [comment]
:+1: . Thanks for fixing.

**Pull request merged**